Cargo.toml

```diff
@@ -18,14 +18,17 @@ keywords = ["msgbox", "gui", "gtk"]
 license = "MIT"
 
 [dependencies]
-thiserror = "1.0"
+thiserror = "1.0.21"
+
+[target.'cfg(target_family = "unix")'.dependencies]
```

Comment:
`glib` crate is not necessary on macOS. The linker might not include unnecessary bits in the output, but I want to avoid fetching and compiling unnecessary dependencies unless I really have to.

Please move `glib` dependency next to `gtk`.